web-app/src/app/screens/Feed/components/FeedSummary.tsx

```diff
@@ -366,7 +366,89 @@ export default function FeedSummary({
               )}
           </Box>
         )}
+      {feed?.related_links != null && feed.related_links?.length > 0 && (
+        <Box sx={boxElementStyle}>
+          <StyledTitleContainer>
+            <LinkIcon />
+            <Typography variant='subtitle1' sx={{ fontWeight: 'bold' }}>
+              {t('relatedLinks')}
+            </Typography>
+          </StyledTitleContainer>
+
+          <Box>
+            <ul
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                paddingLeft: '20px',
+                marginTop: 0,
+                listStyle: 'disc',
+              }}
+            >
+              {feed.related_links.map((relatedLink) => {
+                const url = relatedLink.url;
+                const code = relatedLink.code ?? '';
+                const description = relatedLink.description;
+
+                return (
+                  <li key={url} style={{ marginBottom: 8 }}>
```

Comment:
please specify '8px', marginBottom: 8 can mean different things if in sx. More readable